tools/mypy_wrapper.py

```diff
@@ -36,13 +37,15 @@ def config_files() -> Set[str]:
     return {str(p) for p in Path().glob('mypy*.ini')}
 
 
-def glob(*, pattern: str, filename: PurePosixPath) -> bool:
+def is_match(*, pattern: str, filename: str) -> bool:
     """
     Return True iff the filename matches the (mypy ini) glob pattern.
     """
     return any(
-        fnmatch.fnmatchcase(str(prefix), pattern)
-        for prefix in chain([filename], filename.parents)
+        filename == path or filename.startswith(f'{path}/')
```

Comment:
Will this work on Windows?

Reply:
Oh good point! I meant to write an `.as_posix()` in there but forgot; let me add that